src/GitHub/GraphQL.hs

```diff
@@ -84,16 +88,63 @@ data ParamName
     | ParamName
     | ParamLast
     | ParamStates
+    | ParamOrderBy
+    | ParamField
+    | ParamDirection
     deriving stock (Show)
 
 data ParamValue
+    {- | Textual parameter:
+
+    @
+    name: "github-graphql"
+    @
+    -}
     = ParamStringV !Text
+
+    {- | Integer parameter:
+
+    @
+    last: 2
+    @
+    -}
     | ParamIntV !Int
+
+    {- | Issue/PR states:
+
+    @
+    states: [CLOSED, MERGED]"
+    @
+    -}
     | ParamStatesV !(NonEmpty State)
+
+    | ParamIssueOrderField !IssueOrderField
+
+    | ParamOrderDirection !OrderDirection
+
+    {- | Record parameters:
+
+    @
+    orderBy: {field: CREATED_AT, direction: DESC}
+    @
+    -}
+    | ParamRecordV !(NonEmpty QueryParam)
     deriving stock (Show)
 
+-- TODO: rename to PullRequestState
 data State
     = Open
     | Closed
     | Merged
     deriving stock (Show)
```

Comment:
This is because there's also `IssueState` type that doesn't have the `Merged` state. And, I guess, we will need to add prefixes in the future to avoid disambiguation...